src/fu-engine.c

```diff
@@ -1054,17 +1044,83 @@ fu_engine_check_hardware_requirement (FuEngine *self, AsApp *app, GError **error
 }
 #endif
 
+static GHashTable *fu_engine_get_report_metadata (FuEngine *self);
+
+static gchar *
+fu_engine_requirement_convert_key (const gchar *name)
+{
+	struct {
+		const gchar *old;
+		const gchar *new;
+	} project_names[] = {
+		{ "FwupdVersion",		"org.freedesktop.fwupd" },
+		{ "FwupdateVersion",		"com.redhat.fwupdate" }, //FIXME?
+		{ "EfivarVersion",		"com.redhat.efivar" }, //FIXME?
+		{ "AppstreamGlibVersion",	"org.freedesktop.appstream-glib" },
+		{ "GUsbVersion",		"org.freedesktop.gusb" },
+		{ "KernelVersion",		"org.kernel.linux" },
```

Comment:
I could potentially see kernel version used by several plugins, but I'm worried about how it affects backports (thinking with a RHEL hat on).  I think in general you should leave a comment around it that this should be last resort and rather the plugin should test for the appropriate kernel interface.

Reply:
(or just not allow using this key)

Reply:
I think dropping org.kernel.linux is the best plan; it's useful to see in the report, but not to test against. Fixed.